README.md

```diff
@@ -2,13 +2,27 @@
 
 <p align="center">
   <a href="https://github.com/fulcrumgenomics/fqtk/actions?query=workflow%3ACheck"><img src="https://github.com/fulcrumgenomics/fqtk/actions/workflows/build_and_test.yml/badge.svg" alt="Build Status"></a>
+  <img src="https://img.shields.io/crates/l/fqtk.svg" alt="license">
+  <a href="https://crates.io/crates/fqtk"><img src="https://img.shields.io/crates/v/fqtk.svg?colorB=319e8c" alt="Version info"></a>
+  <a href="http://bioconda.github.io/recipes/fqtk/README.html"><img src="https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat" alt="Install with bioconda"></a>
+  <br>
 </p>
 
 A toolkit for working with FASTQ files, written in Rust.
 
 ## Installing
 
-Until the first public release, the only way to install the tool is by checking out the git repository and building with cargo.
+### Installing with `conda`
```

Comment:
For this and the crates sectoin, can you please provide ~1 sentence and a link to the installer(s) for miniconda and rustup?